src/corehost/common/pal.h

```diff
@@ -180,21 +190,17 @@ namespace pal
 
 #endif
 
-#if defined(__APPLE__)
-    enum os_moniker_t
-    {
-        sierra = 16,
-        el_capitan = 15,
-    };
-
-    bool get_os_moniker(os_moniker_t* moniker);
-#endif
-
     inline void err_flush() { std::fflush(stderr); }
     inline void out_flush() { std::fflush(stdout); }
 
     // Based upon https://github.com/dotnet/core-setup/blob/master/src/Microsoft.DotNet.PlatformAbstractions/Native/PlatformApis.cs
     pal::string_t get_current_os_rid_platform();
+    inline pal::string_t get_current_os_baserid()
```

Comment:
Should we consistently say "base RID" or "fallback RID"?  Here the method is named one thing, but the const is a different thing.